src/Relude/Bifoldable.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE CPP #-}
+
+{- |
+Copyright:  (c) 2018 astynax
+License:    MIT
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+This module reexports some 'Bifoldable' related stuff.
+-}
+
+module Relude.Bifoldable
+       (
+#if MIN_VERSION_base(4,10,0)
+         Bifoldable (..)
+       , bifoldl'
+       , bifoldl1
```

Comment:
Let's not reexport `bifoldl1` and `bifoldr1`. Looks like these functions are partial